src/simple_commander/main.py

```diff
@@ -191,10 +194,13 @@ def __init__(self, x, y, angle, bonus=0, speed=0, life_count=3, type='',
     def decrease_life(self, units):
```

Comment:
Single unit should know nothing about all other units on the game field - it must be in responsibility of GameController only!